src/screens/User/Profile.js

```diff
@@ -146,20 +142,25 @@ class Profile extends Component {
 
 	renderSocialMedia() {
 		const { logoContainer, socialMediaRow } = styles;
-		const { color } = this.props.activeUser;
+		const { color, email, linkedin } = this.props.activeUser;
 
 		return (
 			<View style = {{ flex: 0.2 }}>
 				<View style = {{ flex: 0.03 }} />
 				<View style = { socialMediaRow }>
 					<View style = { [logoContainer, { backgroundColor: color, flex: 1 }] }>
-						<TouchableOpacity onPress = { () => Alert.alert("Coming Soon") }>
+						<TouchableOpacity onPress = { () => {
+							appLinkingHandler(`https://www.linkedin.com/in/${linkedin}`,
+								"No profile has been added. To add one, tap on Edit Profile.");
+						} }>
 							<Ionicons name = "logo-linkedin" size = { height * 0.045 } color = "white" />
 						</TouchableOpacity>
 					</View>
 					<View style = {{ flex: 0.01 }} />
 					<View style = { [logoContainer, { backgroundColor: color, flex: 1 }] }>
-						<TouchableOpacity onPress = { () => Alert.alert("Coming Soon") }>
+						<TouchableOpacity onPress = { () => {
```

Comment:
I'm not 100% sure about this but I think it could all be on 1 line.If it can't then don't worry about this.